src/main/java/training/App.java

```diff
@@ -5,23 +5,18 @@
 package training;
 
 
-import java.util.Map;
-
 public class App {
 
     public String getGreeting() {
         return "Hello consortium world.";
     }
 
     public static void main(String[] args) throws InterruptedException {
+       
 
-            TableCreator tc =new TableCreator();
-
-        String accessId = System.getenv("AWS_ACCESS_KEY_ID");
-        String accessSecret= System.getenv("AWS_SECRET_ACCESS_KEY");
-
-        tc.createTable(accessId,accessSecret);
+ItemsCreator item=new ItemsCreator();
 
+      item.putItem();
```

Comment:
You are not using the return value here ise to wrie system out println.

Reply:
i dont get what u mean here?